src/Plugins/ComplexCore/CMakeLists.txt

```diff
@@ -141,13 +141,16 @@ add_subdirectory(${${PLUGIN_NAME}_SOURCE_DIR}/test)
 #------------------------------------------------------------------------------
 # If there are additional library packages that need to be found, there is where
 # to do that using the usual find_package(....) call
-# find_package(....)
-
+# ------------------------------------------------------------------------------
+# The ExecuteProcess Filter requires the `reproc` library
+# ------------------------------------------------------------------------------
+find_package("reproc" CONFIG REQUIRED)
```

Comment:
```suggestion
```

Reply:
Look for the reproc++ library instead. Fixed